### PR TITLE
Update Flag effect

### PR DESCRIPTION
**Pull request by Cosmicbase**
**Summary**

Added Russian Flag to 'Flag effect'

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of web configuration, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing setups:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's body (e.g. `Fixes: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated (docs/docs/en)
- [ ] Related tests have been updated

**PLEASE DON'T FORGET TO ADD YOUR CHANGES TO CHANGELOG.MD**
- [ ] Yes, CHANGELOG.md is also updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: effects/flag.py [hunk 1]
```diff
@@ -178,6 +178,12 @@ def printFlag(country):
 		hyperion.imageSolidFill(int(iW*0.33), 0, int(iW*0.33), iH, 255, 0, 0)
 		hyperion.imageSolidFill(int(iW*0.66), 0, iW, iH, 255, 255, 0)
 
+        # de flag (Russia)
```
Comment (tpmodding) [inline effects/flag.py:181]: müsste das nicht ru flag heissen?

**Review by tpmodding — CHANGES_REQUESTED** [review]

File: effects/flag.py (hunk 1 above)
Comment (Cosmicbase) [reply effects/flag.py:181]: Ups sorry mein 1. Versuch. Sorry ja müsste ru heißen.

Comment (Cosmicbase) [reply effects/flag.py]: Müsste jetzt passen.😁

Comment (Paulchen-Panther) [reply effects/flag.py]: Sieht gut aus. 👍🏻

**Pull request closed without merging**